Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/contests/types.tsx

```diff
@@ -0,0 +1,51 @@
+interface IContestResultsType {
```

Comment:
This should be in `common` not in `hooks`

Reply:
The file or the types itself should be in the common `types` filte?